working-groups/security-wg.md

```diff
@@ -0,0 +1,55 @@
+* Collaborate with the Presto community to prioritize and address identified security vulnerabilities.
+* Develop and implement security improvements and best practices for the Presto project.
+* Coordinate security training and awareness initiatives for the Presto community.
+* Provide guidance and assistance on security-related questions and concerns from the community.
+
+## Principles
+
+* **Openness** and **Transparency**:
+    * Meetings of the Security WG shall be open to all participants who are dedicated to improving the security of the Presto project.
+    * Decisions of the Security WG will be discussed openly in meetings, mailing list discussions, and/or issue threads.
+* **Neutrality**:
+    * The Security WG should operate in a manner which prioritizes the best interests of the Presto project ecosystem as a whole, and which does not privilege any particular vendor or distributor of Presto-related goods and services.
+    * All activities of the Security WG shall comply at all times with the [Antitrust Policy of The Linux Foundation](/https://www.linuxfoundation.org/antitrust-policy//).
+* **Technical merit**:
+    * Decisions on security improvements and best practices should be based on technical merit, in light of the goals of aligning with the Presto project codebase and fitting with reasonable expectations of end users of Presto.
+    * Participants in discussions should have the opportunity to share, discuss, and advocate for the merit of their views. However, not all views will necessarily be followed or implemented.
+* **Objectivity**:
+    * To the extent possible, decisions should be based on objectively-measurable criteria, rather than subjective considerations requiring personal evaluation and determination.
+
+## Governance
+* **Working Group**:
+    * The TSC is establishing the Security WG as a working group of the TSC, pursuant to section 2(h)(iv) of the Presto Technical Charter.
+* **Chairperson**:
+    * The TSC will appoint an initial Chairperson of the Security WG.
+    * The TSC may, in its discretion, remove the Chairperson and/or appoint a new Chairperson of the Security WG.
+    * The current Chairperson will be listed on the [Security Working Group Github Project](/https://github.com/prestodb/security-wg/).
+    * The Chairperson shall be responsible for:
+        * scheduling regular meetings of the Security WG community;
+        * facilitating open discussion among Security WG community participants;
+        * assisting the Security WG community to seek to reach consensus;
+        * recording decisions that are reached by the Security WG community;
+        * documenting areas of disagreement where consensus cannot be reached, for escalation to the TSC; and
+        * keeping the TSC regularly informed about the status of the Security WG’s efforts.
+    * The Chairperson shall not be responsible for:
+        * deciding the outcome of technical decisions if consensus cannot be reached.
+* **Meetings**:
```

Comment:
Shall we describe the frequency and time for the meetings, and who are required to attend?

Reply:
The chair, once appointed, can self-determine that